src/Routes.js

```diff
@@ -68,6 +71,8 @@ export const Routes = () => {
     }
   }, []);
 
+  const GroupsComponents = edgeGroupsUse ? EdgeGroupsView : InventoryGroups;
```

Comment:
The flag indicates customers that has an enforced edge groups view. Could we rename the flag to better illustrate this fact? Like `enforceEdgeGroups` instead of `edgeGroupsUse`?